cpp/splinepy/py/py_knot_insertion_matrix.hpp

```diff
@@ -0,0 +1,499 @@
+#pragma once
+
+#include <algorithm>
+#include <sstream>
+
+#include "pybind11/numpy.h"
```

Comment:
can we keep `#include <>` format? what happened?

Reply:
Local header files should be included using "" whereas system libraries are to be included with <>. As pybind is in the local path I changed it to "".

Reply:
it's an external header

Reply:
Yes but it's located in the local path (in third_party), this is why I would set it in quotes. <> are for system headers.

Reply:
can we do this is next PR as clean up? this just adds inconsistency for now

Reply:
or leave it and we will just clean up